README.md

```diff
@@ -1,2 +1,26 @@
 # ng-infinite-scroll-api-decorator
+
 Makes it easier to connect paged API responses to ngInfiniteScroll
+
+## Use
+
+To initialize, pass a URL with the token `{page}` which will be replaced as each page is loaded.
+
+```js
+$scope.list = new infiniteScroll( "/users?p={page}" );
+```
+
+The attributes shown here are attributes of ngInfiniteScroll, but the conditionals and callbacks are part of infiniteScrollDecorator.
```

Comment:
So ngInfiniteScroll is an external library?  I guess that wasn't clear to me from the start, maybe put a link to it somewhere in the readme, perhaps just link the reference in that first sentence at the top.  (This is my first time ever looking at this really.)

Would be nice if the "conditionals and callbacks" were actually documented/explained in any capacity.